tests/unittests/HyphenTest.cpp

```diff
@@ -33,6 +33,7 @@
 
 #include <cassert>
 #include <cctype>
+#include <random>
```

Comment:
i think <cassert> and <cctype> can be removed.

Reply:
Thanks. Fixed